test/src/test/java/hudson/model/DirectoryBrowserSupportTest.java

```diff
@@ -149,8 +149,16 @@ public void doubleDots2() throws Exception {
         p.getBuildersList().add(new Shell("mkdir abc; touch abc/def.bin"));
         j.buildAndAssertSuccess(p);
 
-        // can we see it?
-        j.createWebClient().goTo("job/" + p.getName() + "/ws/abc%5Cdef.bin", "application/octet-stream");
+        try (JenkinsRule.WebClient wc = j.createWebClient()) {
```

Comment:
See [JENKINS-73129](https://issues.jenkins.io/browse/JENKINS-73129) for an explanation of this change. Newer versions of the Servlet API are more secure by default and [reject suspicious sequences](https://jakarta.ee/specifications/servlet/6.0/jakarta-servlet-spec-6.0#uri-path-canonicalization) with a 400 error rather than a 404. We appreciate this increased security, but our test must be adapted to expect the new error code on Windows.